ui/app/adapters/transform/base.js

```diff
@@ -42,7 +42,10 @@ export default ApplicationAdapter.extend({
   fetchByQuery(query) {
     const { backend, modelName, id } = query;
     return this.ajax(this.url(backend, modelName, id), 'GET').then(resp => {
-      return resp;
+      return {
+        ...resp,
+        backend,
```

Comment:
This is required to attach the backend to a transform/role (or any other nested transform object)